vault/dendron.topic.links.md

```diff
@@ -1,97 +1,80 @@
+Dendron provides many ways to create links between notes and assets. This range from regular markdown links to extended syntax such as wikilinks, file links, and much more.
 
 ## Features
 
-### Wiki Links
+### Link to various locations with both regular Markdown links and extended link syntax
 
-![[dendron://dendron.dendron-site/dendron.topic.links.wiki-link]]
-
-### File Links
+- Dendron is built on top of Markdown, and regular Markdown links work out of the box. 
+- Furthermore, Dendron extends the syntax of regular Markdown links and support Wikilinks. Many convenient features are built around wikilinks to support various use cases when creating links between notes in your knowledge base.
 
-![[dendron://dendron.dendron-site/tags.stage.sprout]]
+### Manipulate links using various commands
 
-You can link to files in your workspace that are not notes (or not in a vault) using wikilinks. The
-easiest way to link to a file is using the [[Copy Note Link|dendron.ref.commands#copy-note-link]]
-command, which will automatically create a link for you. As with note files, copy note link
-will create a [[Block Anchor|dendron://dendron.dendron-site/dendron.topic.note-reference#block-anchor]] for you if you have a region of text selected.
+- Linking is a powerful feature that helps you grow your knowledge base. Dendron provides various ways facilitate this through commands. You can easily create, copy, and convert your links in your notes using these commands so you can focus on what's important.
 
-Otherwise, you need to write inside a wikilink the relative path to the file
-from where your `dendron.yml` file is located. For example, if the root of your
-workspace contains `dendron.yml` and a `src` folder, you can write
-`[[src/index.js]]`. Alternatively, if you are linking to a file inside the
-`assets` folder of a vault, you can simply type `[[assets/example.py]]` to link to it.
+### Navigating easily through linked notes
 
-### Children Links
+- Once links are created, you can explore your knowledge base in an organic way. Dendron provides various ways you can navigate through linked notes.
 
-When you view notes using the [[preview|dendron://dendron.dendron-site/dendron.topic.preview]] or the [[published|dendron.topic.publish]] site, Dendron will auto-generate a `Children` section at the end of the note.
+### Visualize links
 
-![Children](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/Quickstart_-_Dendron.jpg)
+- Dendron provides a way to visualize the links between your notes. See [[Graph View|dendron://dendron.dendron-site/dendron.topic.graph-view]] for more information.
 
-- TIP: this can be turned off using the [[nav_exclude_children|dendron://dendron.dendron-site/dendron.topic.publish.config.nav_exclude_children]] option (publishing only)
+## Components
 
-### Backlinks
-![[dendron://dendron.dendron-site/dendron.topic.links.backlink]]
+### Wiki Links
```

Comment:
all other compoenents have a `Summary` and `Getting Started` whereas `Wiki Links` has `Summary` and `Features`. would make this consistent. for the overview, I would just show the summary of each component vs the entire note